.github/CODEOWNERS

```diff
@@ -75,6 +75,7 @@ airflow/providers/cncf/kubernetes/executors/ @dstandish @jedcunningham
 /tests/system/providers/amazon/ @eladkal @o-nikolas
 /tests/providers/common/sql/ @eladkal
 /tests/providers/slack/ @eladkal
+/tests/providers/teradata/ @SatishChGit
```

Comment:
> Who can use this feature
People with write permissions for the repository can create or edit the CODEOWNERS file and be listed as code owners

You cannot be a code owner if you don't have write access to the repository

```suggestion
```

Reply:
@SatishChGit

Reply:
> > Who can use this feature
> > People with write permissions for the repository can create or edit the CODEOWNERS file and be listed as code owners
> 
> You cannot be a code owner if you don't have write access to the repository

Removed this entry.